lib/MusicBrainz/Server/Entity/LinkAttributeType.pm

```diff
@@ -35,7 +35,7 @@ has 'root' => (
 sub l_name {
     my $self = shift;
     my $rootid = defined $self->root ? $self->root->id : $self->root_id;
-    if ($rootid == $INSTRUMENT_ROOT_ID) {
+    if (defined $rootid && $rootid == $INSTRUMENT_ROOT_ID) {
```

Comment:
I'd like to understand which pages are triggering this too (I'm guessing edit pages again) - if it's not something we can fix, a short comment here describing when this happens would be nice.